-------------------------------------------------------------------------------
Test set: ru.practicum.shareit.exceptionTest.NotFoundExceptionTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in ru.practicum.shareit.exceptionTest.NotFoundExceptionTest
